dolt_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Introducing Dolt + Metaflow
+---
+
+## Background
+This post details how to use Metaflow with Dolt. [Metaflow](https://metaflow.org/) is a workflow manager that offers data scientists the ability to define local experiments and scale those experiments to production jobs from a single API. [Dolt](https://docs.dolthub.com/) is a version controlled relational database. It provides a familiar SQL interface along with Git-like version control features for capturing historical states. Each commit corresponds to a complete state of the database at the time the commit was created. Both Dolt and Metaflow are open source.
+
+To illustrate the power of this integration we use an example of a Metaflow piepline that produces a final result as a Pandas data-frame that is stored in a database for use by application layer services:
+![]()
+
+This blog post will breakdown how to use Dolt to store final and intermediate results to augment Metaflow based pipelines with full reproducibility, lineage, and back-testing capabilities. Dolt allows users to see where their data came from, what it looked like at every transformation, and to feed historical versions of it into new jobs.
```

Comment:
remove "to store final and intermediate results"

Reply:
if you want to say that put it in a different sentence. It complicates the lede.